addons/website/static/src/builder/plugins/options/ecomm_categories_showcase_option.xml

```diff
@@ -0,0 +1,66 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates xml:space="preserve">
+
+<t t-name="html_builder.EcommCategoriesShowcaseOption">
+    <BuilderRow label.translate="# of Blocks">
+        <BuilderSelect action="'blockCount'">
+            <BuilderSelectItem actionValue="'2'">2</BuilderSelectItem>
+            <BuilderSelectItem actionValue="'3'">3</BuilderSelectItem>
+            <BuilderSelectItem actionValue="'4'">4</BuilderSelectItem>
+        </BuilderSelect>
+    </BuilderRow>
+    <BuilderRow label.translate="Layout">
+        <BuilderSelect classAction="''">
+            <BuilderSelectItem classAction="'first-large-layout'">First block larger</BuilderSelectItem>
+            <BuilderSelectItem classAction="'equal-layout'">Equal size</BuilderSelectItem>
+        </BuilderSelect>
+    </BuilderRow>
+</t>
+
+<t t-name="html_builder.EcommCategoriesShowcaseBlocksDesign">
+    <hr class="mx-2"/>
+    <div class="px-2 fs-6">Blocks Design</div>
+    <div class="px-2 opacity-75 small mb-2">Global options for all inner blocks.</div>
+    
+    <BuilderRow label.translate="Min. Height">
+        <BuilderRange 
+            action="'setMinHeight'" 
+            actionParam="['0', '1', '2']"
+            max="2" 
+            step="1"/>
+    </BuilderRow>
+
+    <BuilderRow label.translate="Shadows">
+        <BuilderCheckbox 
+            classAction="'shadow'" 
+            applyTo="'.s_ecomm_categories_showcase_block'"
+            preview="false"/>
+    </BuilderRow>
+    
+    <BuilderRow label.translate="Space Out">
+        <BuilderCheckbox 
+            id="'spacing_opt'"
+            action="'spacingToggle'"/>
+    </BuilderRow>
+    <BuilderRow label.translate="Roundness" t-if="isActiveItem('spacing_opt')">
+        <BuilderRange 
+            action="'setClassRange'" 
+            actionParam="['rounded-0','rounded-1','rounded-2','rounded-3','rounded-4','rounded-5']" 
+            applyTo="'.s_ecomm_categories_showcase_block'"
+            max="5" 
+            step="1"/>
+    </BuilderRow>
```

Comment:
Ok that's annoying... but maybe there is not another way to do this at the moment. Can you just add a comment on top of this: `// TODO maybe use CompositeAction`, we'll see with the OWL team to improve.